crates/oxc_codegen/src/lib.rs

```diff
@@ -569,13 +570,13 @@ impl<'a> Codegen<'a> {
 
     fn add_source_mapping(&mut self, position: u32) {
         if let Some(sourcemap_builder) = self.sourcemap_builder.as_mut() {
-            sourcemap_builder.add_source_mapping(&self.code, position, None);
+            sourcemap_builder.add_source_mapping(self.code.as_ref(), position, None);
```

Comment:
Maybe add an `as_bytes` method to do the same? `code.as_ref()` is a slightly unintuitive/mysterious (what kind of ref?)

Reply:
I've made this change in an additional commit.